src/corpus/domain/chapter.ts

```diff
@@ -159,16 +159,33 @@ export class ChapterDisplay {
     return this.textHasDoi ? textIdToDoiString(this.id.textId) : ''
   }
 
+  mapNames(
+    names: readonly Author[] | readonly Translator[]
+  ): { family: string[]; given: string[] }[] {
+    return names.map((_name) => {
```

Comment:
Is there a reason for using the `_` prefix? AFAIK, per convention that indicates unused parameters (and suppresses the "unused parameter" error), but that doesn't apply here.

Reply:
Come to think of it, the `Author` and `Translator` interfaces are quite similar, and the logic here is specific to them. Maybe it would be worth it to change them into classes and add `family` and `given` getters, as well as the `nameToString` function below as method?